content/_code-samples/issue-a-token/js/list-account-tokens.js

```diff
@@ -0,0 +1,59 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
```

Comment:
I also see a mix of single & double quote string literals being used. Can you stick to using one?

Reply:
Thanks, just noticed that